apps/site/util/downloadUtils/index.tsx

```diff
@@ -0,0 +1,215 @@
+      !supportsVersion(item.compatibility) ||
+      !supportsRelease(item.compatibility),
+  }));
+};
+
+// Here the list of Operating System Dropdown items are defined !
+export const OPERATING_SYSTEMS: Array<DownloadDropdownItem<UserOS>> = [
+  {
+    label: osConstants[0].label,
+    value: osConstants[0].value as UserOS,
+    compatibility: osConstants[0].compatibility,
+    iconImage: <OSIcons.Microsoft width={16} height={16} />,
+  },
+  {
+    label: osConstants[1].label,
+    value: osConstants[1].value as UserOS,
+    compatibility: osConstants[1].compatibility,
+    iconImage: <OSIcons.Apple width={16} height={16} />,
+  },
+  {
+    label: osConstants[2].label,
+    value: osConstants[2].value as UserOS,
+    compatibility: osConstants[2].compatibility,
+    iconImage: <OSIcons.Linux width={16} height={16} />,
+  },
+  {
+    label: osConstants[3].label,
+    value: osConstants[3].value as UserOS,
+    compatibility: osConstants[3].compatibility,
+    iconImage: <OSIcons.AIX width={16} height={16} />,
+  },
+];
+// Here the list of Install Method Dropdown items are defined !
+
+export const INSTALL_METHODS: Array<
+  DownloadDropdownItem<Types.InstallationMethod> &
+    Required<
+      Pick<DownloadDropdownItem<Types.InstallationMethod>, 'info' | 'url'>
+    >
+> = installMethods.map(method => {
+  // Map compatibility.os from string[] to UserOS[] if present
+  // Map compatibility.releases from string[] to NodeReleaseStatus[] if present
+  const compatibility = {
+    ...method.compatibility,
+    os: method.compatibility?.os?.map(os => os as UserOS),
+    releases: method.compatibility?.releases?.map(
+      (release: string) => release as NodeReleaseStatus
+    ),
+  };
+  return {
+    ...method,
+    key: method.key as Types.InstallationMethod,
+    value: method.value as Types.InstallationMethod,
+    // optionally override the label if needed by using InstallationMethodLabel from JSON
+    label:
+      InstallationMethodLabel[
+        method.key as keyof typeof InstallationMethodLabel
+      ] || method.label,
+    iconImage:
+      installMethodIconsMap[method.value as keyof typeof installMethodIconsMap],
+    info: method.info as IntlMessageKeys, // cast to expected type
+    compatibility,
+  };
+});
+
+// Here the list of Package Manager  Dropdown items are defined !
+
+export const PACKAGE_MANAGERS: Array<
+  DownloadDropdownItem<Types.PackageManager>
+> = [
+  {
+    label: PackageManagerLabel.NPM,
+    value: 'NPM',
+    compatibility: {},
+    iconImage: <PackageManagerIcons.NPM width={16} height={16} />,
+  },
+  {
+    label: PackageManagerLabel.YARN,
+    value: 'YARN',
+    compatibility: { semver: ['>= v14.19.0', '>= v16.9.0'] },
+    iconImage: <PackageManagerIcons.YARN width={16} height={16} />,
+  },
+  {
+    label: PackageManagerLabel.PNPM,
+    value: 'PNPM',
+    compatibility: { semver: ['>= v14.19.0', '>= v16.9.0'] },
+    iconImage: <PackageManagerIcons.PNPM width={16} height={16} />,
+  },
+];
```

Comment:
Can these go in the JSON?

Reply:
Whoops ! 

Everytime I'm missing something or the other ! 

Sorry ,  On it !